libs/designsystem/src/lib/components/avatar/avatar.component.ts

```diff
@@ -12,6 +14,9 @@ export class AvatarComponent implements OnInit {
   @Input() shadow: boolean;
   @Input() text: string;
   @Input() overlay: boolean;
+  @HostBinding('class')
+  @Input()
+  themeColor: MainColor | 'semi-light';
```

Comment:
`$main-colors` is tagged with a deprecation in `_colors.scss` so consider using plain string literal type (as in Flag component)